uber/templates/macros.html

```diff
@@ -82,6 +82,36 @@
 {%- endmacro %}
 
 
+{% macro form_group_scaffolding(
+    label=False,
+    help=False,
+    is_required=False,
+    size=6,
+    form_class='',
+    static_wrap=False) -%}
```

Comment:
Oh yea, I did notice that. That makes sense! PyCharm's settings were defaulted to 4 spaces, but I'll change that for HTML files and fix this file's indents up.